### DISCUSSION TITLE
Help exporting my wallet in Win 10

### DISCUSSION THREAD

**Discussion by weekly-natural** [Chia-Network/chia-blockchain#1041] [Q&A] [answered]
I was farming CHIA using version 0.1.27 but the version had some issues with stability. I downloaded the latest 0.2.2 version and logged in using the passcodes for my account. However, after synchronizing I did not see the already farmed CHIA on my account but could import an earlier wallet. I can still start the previous version and see the farmed tokens on my account but don't really know how to export the wallet into a file.

I found this guide but not sure how to utilize this. I'm running a Windows 10 environment and part of the guide is also pretty specific to Mac. Is anyone able to assist with this? Much appreciated!

**Comment by hoffmang9** [comment, marked as answer]
The RC1/2 chain is a new chain and was reset. TXCH doesn't persist over new chains.

**Comment by NickJavens** [comment]
I had the same issue.  Sorry hoffmang9 I don't understand the response, I am a bit of a novice at this.  Does this mean that the TCXH previously won are not able to be added to the wallets in the latest build 0.2.2?

**Comment by hoffmang9** [comment]
As it says in the release notes, we started a new blockchain from scratch for RC1/2. Since your test XCH (TestXCH) winnings are on the blockchain - they don't get migrated to any new chain we start. Test XCH are likely to be deleted often. Only mainnet XCH will persist.

**Reply by narendraj9** [reply]
Will the plot files remain relevant in the mainnet or the farmers would need to plot them again?

**Reply by hoffmang9** [reply]
Plots k32 or larger, and keys always come forward on each release and are mainet ready.
